ui/app/js/services/emergency_shutoffs.js

```diff
@@ -0,0 +1,54 @@
+angular.module('app').factory('EmergencyShutoffs', function($http, ScheduledChanges) {
+  var service = {
+    get: function() {
+      return $http.get('/api/emergency_shutoff');
+    },
+    create: function(product, channel) {
+      data = {product, channel};
```

Comment:
I think this is the source of your test failure - it doesn't look like valid syntax to me (I'm surprised it's not breaking my manual tests, in fact).

Reply:
Ah, I got it. This is the ES6 shorthand syntax.